lineage/figures/figure6B.py

```diff
@@ -0,0 +1,65 @@
+""" This file contains functions for plotting the performance of the model for censored data. """
+
+import numpy as np
+
+from ..Analyze import run_Analyze_AIC
+from ..LineageTree import LineageTree
+from ..data.Lineage_collections import Gemcitabine_Control
+
+# States to evaluate with the model
+from ..states.StateDistributionGaPhs import StateDistribution
+from .figureCommon import getSetup, lineage_good_to_analyze, subplotLabel
+from .figureS10 import run_AIC, figure_maker
+
+
+desired_num_states = np.arange(1, 8)
+
+
+def makeFigure():
+    """
+    Makes figure 10.
+    """
+    ax, f = getSetup((13.333, 3.333), (1, 4))
```

Comment:
This should just be one figure. The x-axis is the number of states to be picked, and the y-axis is the AIC value as the model tries to find that many states. Why are four subplots needed?

Reply:
Are there 4 data sets?

Reply:
Never mind, I just noticed that you want this to be on master before investigating further.